apps/api/app/services/device_token_service.py

```
@@ -0,0 +1,185 @@
+        """
+        try:
+            result = await self.collection.delete_one({"token": token})
+
+            if result.deleted_count > 0:
+                logger.info(f"Unregistered device token: {token}")
+                return True
+            else:
+                logger.warning(f"Device token not found: {token}")
+                return False
+
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)